courses/models.py

```diff
@@ -372,6 +374,35 @@ def text_id(self):
     def __str__(self):
         return self.title
 
+    def clean(self):
```

Comment:
can we use https://docs.djangoproject.com/en/2.2/ref/validators/ ?

Reply:
We could, however the main use of validators is to extract the validation logic into a standalone unit so that it can be reused elsewhere. Seeing as we don't need this validation anywhere else, it doesn't make a very strong case to extract it for resuse.